11/umbraco-forms/editor/creating-a-form/creating-a-contact-us-form.md

```diff
@@ -0,0 +1,265 @@
+# Creating a Contact Us Form
+
+In this article, we'll look at creating a Contact Us Form.
+
+## Video Tutorial
+
+{% embed url="https://www.youtube.com/watch?ab_channel=UmbracoLearningBase&v=TWLqt-jVdyQ" %}
+How to use Language Variants in Umbraco
+{% endembed %}
+
+## Creating a Composition
+
+To create a Composition, follow these steps:
```

Comment:
Perhaps a little more context here, as to what will be covered in this step? What is a composition, fx, and why do we need it to create a new Form?